tests/test_set_ocean_surface_temperature.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import os
+import numpy as np
+import fv3gfs.wrapper
+import fv3gfs.util
+from mpi4py import MPI
+from util import (
+    get_default_config,
+    get_state_single_variable,
+    main,
+    replace_state_with_random_values,
+)
+
+
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def mask_non_ocean_values(field):
+    is_ocean = np.isclose(get_state_single_variable("land_sea_mask"), 0.0)
+    return np.where(is_ocean, field, np.nan)
+
+
+class PrescribeSSTTests(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(PrescribeSSTTests, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        MPI.COMM_WORLD.barrier()
+
+    def test_prescribing_sst_changes_model_state(self):
+        checkpoint_state = fv3gfs.wrapper.get_state(fv3gfs.wrapper.get_restart_names())
+
+        fv3gfs.wrapper.step()
+        air_temperature_from_default_ocean_temperature = get_state_single_variable(
```

Comment:
Thinking about the code, I'm not sure what gets put in by default when you're not saying to use climatological SST. Is this a "default" value or is it best described as "uninitialized"? If you know, you may want to add a code comment here saying what the default behavior is (with caveats about dependence on the namelist).

Reply:
Added comment explaining what happens in this case.  Indeed it's a little non-obvious.